aspnetcore/fundamentals/minimal-apis/10.0-samples/MinApiIproblemDetailsService/Program.cs

```diff
@@ -0,0 +1,58 @@
+// <snippet_register_IProblemDetailsService_implementation>
+using System.ComponentModel.DataAnnotations;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddProblemDetails(options =>
+{
+    options.CustomizeProblemDetails = context =>
+    {
+        if (context.ProblemDetails.Status == 400)
+        {
+            context.ProblemDetails.Title = "Validation error occurred";
+            context.ProblemDetails.Extensions["support"] = "Contact support@example.com";
+            context.ProblemDetails.Extensions["traceId"] = Guid.NewGuid().ToString();
+        }
+    };
+});
+// </snippet_register_IProblemDetailsService_implementation>
+
```

Comment:
Need to add validation.
```suggestion

builder.Services.AddValidation();

```